app/components/shared/Table/Table.tsx

```diff
@@ -127,22 +127,30 @@ export const Table = <T extends IdObj>({
               </tr>
             </thead>
             <tbody>
-              {paginatedData.map((item) => (
-                <tr key={item.id}>
-                  {Object.entries(removeIdFromObject(item)).map(([key, value]) => (
-                    <td key={key}>
-                      {typeof value === "object"
-                        ? (
-                            value as {
-                              value: string
-                              element: JSX.Element
-                            }
-                          ).element
-                        : value}
-                    </td>
-                  ))}
+              {paginatedData.length ? (
```

Comment:
This condition should be more explicit, like `paginatedData.length > 0` or something like that.

Reply:
added to the last commit 🫡